### DISCUSSION TITLE
People posters

### DISCUSSION THREAD

**Discussion by joca879** [meisnate12/Plex-Meta-Manager#1038] [Q&A]
Hello!

I've found this amazing project and begin to use it with the default collections and overlays...

I've tried the actor.yml to create the cast collections, everything was created as expected... 

Nonetheless I noticed that there isn't bw posters, in the same style, in the Plex-Meta-Manager-Images for the following actors: Henry Cavill, Jason Biggs and Paula Patton.

Can someone create those or tell me where to find them?

Thanks

**Comment by bullmoose20** [comment]
added. check again
